certification/internal/engine/engine.go

```diff
@@ -75,6 +76,7 @@ func (c *CraneEngine) ExecuteChecks(ctx context.Context) error {
 				authn.WithDockerConfig(c.Config.DockerConfig()),
 			),
 		),
+		retryOnceAfter(5 * time.Second),
```

Comment:
Why wouldn't we just use what crane has built in for this by passing in custom options? They already do exponential backoff by default, it would just be changing the default params like in the issue.

Reply:
I was reading this PR wrong since it wasn't showing the `options` that we were already defining. This looks fine to me.